bitbots_vision/src/bitbots_vision/vision_modules/color.py

```diff
@@ -13,13 +13,17 @@
 from .debug import DebugPrinter
 
 
-class ColorDetector:
+class ColorDetector(object):
+    """
+    ColorDetector is abstract super-class of specialized sub-classes.
+    ColorDetectors are used e.g. to check, if a pixel matches the defined color space
+    or to create masked binary images.
+    """
+
     def __init__(self, debug_printer):
         # type: (DebugPrinter) -> None
         """
-        ColorDetector is abstract super-class of specialized sub-classes.
-        ColorDetectors are used e.g. to check, if a pixel matches the defined color space
-        or to create masked binary images.
+        Initiating of ColorDetector.
```

Comment:
Initialization of ColorDetector

Reply:
closed with 7398d9f2ebe6647e5caa15e9846bfb3e33624e37